packages/lit-html/src/lit-html.ts

```diff
@@ -505,14 +521,16 @@ class Template {
               });
               bindingIndex += statics.length - 1;
             } else if (name === marker) {
-              (node as Element).removeAttribute(name);
-              i--;
+              attrsToRemove.push(name);
               this._parts.push({
                 _type: ELEMENT_PART,
                 _index: nodeIndex,
               });
             }
           }
+          for (const name of attrsToRemove) {
```

Comment:
Is mutating attributes while iterating them only bad on IE? Add comment to that effect?

Reply:
Justin covered in his PR